### ISSUE TITLE
implement network sandboxing

### ISSUE THREAD

**Issue by korli** [haikuports/haikuporter#174]
a number of language specific package installers like to download missing dependencies.
It's obviously a no-go, that should be enforced with network sandboxing.

libnetwork could be replaced with dummy library. Any alternatives?

**Comment by waddlesplash** [comment]
Yes, revert this commit: https://github.com/haikuports/haikuporter/commit/f540faa2a00dbc5941f8a9e212fd7d2573823601

Without it the DNS resolver errors when it tries to do anything.
